src/accelerate/inference.py

```diff
@@ -111,9 +115,10 @@ def prepare_pippy(
     example_args=(),
     example_kwargs: Optional[Dict[str, Any]] = None,
     num_chunks=None,
+    gather_output=False,
```

Comment:
let's add the type of each arg since you did it in the docstring and for example_kwargs